webapp/channels/src/components/profile_popover/profile_popover.test.tsx

```diff
@@ -318,11 +321,47 @@ describe('components/ProfilePopover', () => {
         expect(button).toBeDisabled();
     });
 
-    test('should not show the start call button when callsChannelState.enabled is false', async () => {
-        (Client4.getCallsChannelState as jest.Mock).mockImplementationOnce(async () => ({enabled: false}));
+    test('should not show start call button when calls in channel have been explicitly disabled', async () => {
+        const [props, initialState] = getBasePropsAndState();
+        (initialState as any)['plugins-com.mattermost.calls'].channels = {dmChannelId: {enabled: false}};
+
+        renderWithPluginReducers(<ProfilePopover {...props}/>, initialState);
+        expect(await screen.queryByLabelText('Start Call')).not.toBeInTheDocument();
+        expect(await screen.queryByLabelText('Call with user is ongoing')).not.toBeInTheDocument();
+    });
+
+    test('should not show start call button for users when calls mode is on', async () => {
```

Comment:
I'm not sure what you mean by `calls mode is on`? Test mode? (same for others below)

Reply:
Yeah, it's a typo. Will fix.